app/src/components/goals/AddGoal.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <form @submit.prevent="handleSubmit">
+    <label>Goal </label>
+    <input v-focus v-model="goal.title" required >
+    <label>Start Date</label>
+    <input v-focus v-model="goal.startDate" required >
```

Comment:
```suggestion
    <input v-focus v-model="goal.startDate.substring(0, 10)" required >
```
Quick format of date on the front end. Could be done on back end as well in SQL (see goals.js in server)